doc/use_and_config_progress_bar.md

```diff
@@ -0,0 +1,139 @@
+    
+        ![](assets/unsigned_msi_install1.png){:height="50%" width="50%"}
+        Please click **More info** and you will see the following information:
+        
+        ![](assets/unsigned_msi_install2.png)
+        Click **Run anyway** to continue.
+        
+        You may be asked to allow the app because it is unknown publisher as shown below. Please click **Yes**
+        to continue.
+        ![](assets/user_account_control.png)
+        
+    When you finish the installation, run `az --version` and you could see the following information:
+    
+    ```                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
+    > az --version                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
+    azure-cli                         2.17.0
+    
+    core                              2.17.0
+    telemetry                          1.0.6
+ 
+    Python location 'C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\python.exe'
+    ``` 
+    Note: Please make sure your python location is `C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\python.exe` 
+
+2. Using pip (All platform including Windows, Linux, Mac)
+    - `pip install https://zuhdefault.blob.core.windows.net/cliext/azure_cli-2.17.0.post20201230141142-py3-none-any.whl`
+    You would see the following information:
+    ```
+    Collecting azure-cli==2.17.0.post20201230141142 from https://zuhdefault.blob.core.windows.net/cliext/azure_cli-2.17.0.post20201230141142-py3-none-any.whl
+      Using cached https://zuhdefault.blob.core.windows.net/cliext/azure_cli-2.17.0.post20201230141142-py3-none-any.whl
+    ...
+    Successfully installed azure-batch-10.0.0 azure-cli-2.17.0.post20201230141142 azure-mgmt-appconfiguration-1.0.1 azure-mgmt-compute-18.0.0 azure-mgmt-containerregistry-3.0.0rc16 azure-mgmt-network-17.0.0 azure-mgmt-security-0.6.0
+    ```
+    
+    - `pip install https://zuhdefault.blob.core.windows.net/cliext/azure_cli_core-2.17.0.post20201230141142-py3-none-any.whl`
```

Comment:
This won't work as we are releasing the edge build with a `pip` custom index.

Reply:
I see. But because it will not be included in our package, I will keep it here for guidance and remove it when send PR to dev branch.